docs/docs/k8s/ingress.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Ingress Controller
+lang: en-US
+sidebarDepth: 1
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy oidc kubernetes Ingress reverse-proxy
+---
+
+# Kubernetes Ingress Controller
+
+Use Pomerium as a first-class secure-by-default Ingress Controller. Dynamically provision routes from Ingress resources and set policy based on annotations. By defining routes as Ingress resources you can independently create and remove them from Pomerium's configuration. This enables workflows more native to Kubernetes environments such as actions based on pull requests.
+
+## Prerequisites
+
+- A certificate management solution. If you do not already have one in place, this article covers using Cert Manager.
+- A Redis backend with high-persistence is highly recommended.
+
+### System Requirements
+
+- Kubernetes v1.19.0+
+- Pomerium [Helm Chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) v25.0.0+
+
+### Limitations
+
+::: warning
+
+Only one Ingress Controller instance/replica is supported per Pomerium cluster.
+
+:::
+
+## Installation
+
+### Helm
+
+Our instructions for [Installing Pomerium Using Helm](/docs/k8s/helm.md) includes the Ingress Controller as part of the documented configuration.
+
+
+```yaml
```

Comment:
What am I supposed to do with this yaml block in relation to the helm config?

Reply:
Nothing. This is the part of the Helm config in the linked guide. I'm adjusting the copy to make this clearer.